Yoast/javascript
# Fro 93 update styling reactified buttons

## Summary


This PR can be summarized in the following changelog entry:

* Fix some styling issues with the buttons.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Pull this branch
* Go into the `apps/components` folder in terminal/console
* Run `yarn install & yarn start` in that folder
* Browse to `http://0.0.0.0:3333/`
* Click on the `Reactified components` in the top menu
* Check if the arrow stays right in the button when you change the language direction
* Check if a button doesn't push down content if you click on it
* Check if the standard focus from browsers doesn't show up if you focus a button

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-93
